### DISCUSSION TITLE
Package maintanence

### DISCUSSION THREAD

**Discussion by FutureKode** [react-dropzone/react-dropzone#1360] [General]
Hello, is react-dropzone still maintained?

**Comment by mikejoyceio** [comment]
I have the same question. There have been no releases for ~2 years now.

**Reply by rxmarbles** [reply]
The package is still maintained, though it only has a few maintainers with other jobs (speaking as one of said maintainers 😄 ). I'll do my best in the coming days to address issues/PR's.

**Comment by rolandjitsu** [comment]
We haven't been actively maintaining this package, but it should still work for most users. There are a lot of issues open, though, so we should probably start addressing some of those and try to merge/close PRs.
